scenarios/centos-9/edpm_ci.yml

```diff
@@ -4,16 +4,10 @@ cifmw_installyamls_repos: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-o
 cifmw_install_yamls_vars:
   DATAPLANE_REPO: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/dataplane-operator"
   STORAGE_CLASS: crc-csi-hostpath-provisioner
+  BMO_SETUP: false
 
-cifmw_use_crc: true
-cifmw_use_libvirt: true
-cifmw_rhol_crc_use_installyamls: true
-
-pre_infra:
-  - name: Download needed tools
-    inventory: "{{ cifmw_installyamls_repos }}/devsetup/hosts"
-    type: playbook
-    source: "{{ cifmw_installyamls_repos }}/devsetup/download_tools.yaml"
+cifmw_use_crc: false
```

Comment:
hmm maybe. it's used internally, isn't it? We may split things a bit more in order to get a more modular approach with the various moving parts. And avoid duplication, since the more duplication we have, the harder it is to maintain and evolve.

Reply:
@pablintino we may want to create a new "nested.yml" scenario file with the needed pieces in it, and include it in the "old" job definition? That might allow an easier management in the future.

Reply:
it should be better now.